pkg/cosign/env/env.go

```diff
@@ -75,6 +76,11 @@ var (
 			Expects:     "1 if experimental features should be enabled (0 by default)",
 			Sensitive:   false,
 		},
+		VariableOCIExperimental: {
+			Description: "enables experimental cosign features for OCI (1.1+)",
```

Comment:
my preference is to make this a more specific flag

rather than "all experimental OCI features" it should be "experimental support for OCI referrers"

lumping *many* experiments under the same flag is why the cosign 2.0 effort is taking so long 😄